internal/policy/source/source.go

```diff
@@ -67,7 +68,10 @@ type PolicyUrl struct {
 
 // GetPolicies clones the repository for a given PolicyUrl
 func (p *PolicyUrl) GetPolicy(ctx context.Context, workDir string, showMsg bool) (string, error) {
+	// Trim scheme from repo url as this breaks go-getter.
 	sourceUrl := p.PolicyUrl()
+	sourceUrl = strings.TrimPrefix(sourceUrl, "https://")
```

Comment:
We support fetching policies via HTTP as well, so this will break that.

Reply:
Ack. From discussion with @joejstuart, the issue was only seen when the URL had an `https://` scheme with no prepended `git::`. When a URL beginning with `github.com` was used, there was no issue.